tox.ini

```diff
@@ -4,7 +4,7 @@ envlist = py26, py27
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/prestoadmin
-commands = nosetests {posargs}
+commands = nosetests --with-timer --timer-ok 60s --timer-warning 300s {posargs}
```

Comment:
Currently a lot are yellow (lasts more that 60s) and some (1 per 10) are red (lasts more than 300s). 

I didn't want to align this numbers with what we have, but this is what I think we would like to have ;)